AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.spec.ts

```diff
@@ -1289,5 +1290,21 @@ describe('BookingsListComponent', () => {
             component.onChangeNoAllocated();
             expect(bookingPersistService.selectedUsers.length).toEqual(count);
         });
+
+        it('should not show allocated to if work allocation feature flat is off', async () => {
+            launchDarklyServiceSpy.flagChange.next({ 'vho-work-allocation': false });
+            await component.ngOnInit();
+            fixture.detectChanges();
+            const divTohide = fixture.debugElement.query(By.css('allocated-to'));
```

Comment:
variables should be camel-case divToHide

Reply:
made the changes